oxidation/crates/parsec_api_protocol/schema/authenticated.json

```diff
@@ -0,0 +1,1996 @@
+{
```

Comment:
this must be splitted into multiple json files (one per cmd), otherwise it's just too much pain to work with such a big file :smile:

Reply:
Ok I will try something

Reply:
Done

Reply:
:+1: